jsapp/scss/components/_kobo.multi-checkbox.scss

```diff
@@ -0,0 +1,18 @@
+.multi-checkbox {
```

Comment:
You need to add `@import "./components/kobo.multi-checkbox";` somewhere in `jsapp/scss/main.scss`